src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -549,7 +552,9 @@ protected override void Dispose(bool disposing)
                 state,
                 CancellationToken.None,
                 TaskCreationOptions.DenyChildAttach,
-                TaskScheduler.Default);                
+                TaskScheduler.Default);
+
+            tcs.Task.ContinueWith(t => LogResponse(t, loggingRequestId));
```

Comment:
This has allocations and should be done tracing is enabled. Further, it causes a closure that results allocations, so the fix needs take that into account.